frontend/src/views/web/permission-denied.html

```diff
@@ -3,15 +3,20 @@
         <div class="col s12 m6 offset-m3">
             <div class="card horizontal ev-md-container ev-card-panel ev-z-depth-5 ">
                 <div class="card-image">
-                    <img src="dist/images/email.png" height="100">
+                    <img src="dist/images/email.png" height="135">
                 </div>
                 <div class="card-stacked">
                     <div class="card-content">
-                        <p><strong>Unable to create or select a team.</strong>
-                            <br> {{perm.emailError}}</p>
+                        <p><strong>Unable to create or select a team.</strong><br>
+                        Hi {{perm.user.username}}!
```

Comment:
Please remove the <br> tag and change the whole message to 
"Hi <XYZ>, it looks like you have not verified your email address. A verification link has been sent to <email_address@xyz.com>. In order to host or participate in a challenge, please verify your email address by clicking on the verification link in the email."

Reply:
Just to add to it, it would be nice if we can have the feature of "Resend Email Verification" and add a link to that here.